src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -755,4 +767,19 @@ protected function catchKnownExceptions($throwable)
 
         return parent::catchKnownExceptions($throwable);
     }
+
+    public static function splitStreetNumber(string $streetNumber): array
+    {
+        if (preg_match('/^(?P<streetnumber>\d*+)-?(?P<flatnumber>[A-Za-z\d]+|[A-Za-z\d-]+)$/i', $streetNumber, $matches)) {
```

Comment:
```suggestion
        if (preg_match('/^\s*(?P<streetnumber>\d+)\s*-?\s*(?P<flatnumber>[A-Za-z\d]+|[A-Za-z\d-\s*]+)$/i', $streetNumber, $matches)) {
```
Add handling of spaces. Please add to tests as well.

Reply:
Changes implemented with a little bit of modification with the suggestions, also added `/` in regex.